TRB_label_assist/models.py

```diff
@@ -1049,7 +1071,7 @@ class EnsembleClassifier(TripClassifier, metaclass=ABCMeta):
     replaced_predictor = NotImplemented
 
     # required methods
-    def fit(self, train_df):
+    def fit(self, train_df,ct_entry=None):
```

Comment:
I would rename to `unused` to clarify that we don't use `ct_entry` here, or add a code comment if you want to use a named parameter instead of a positional parameter.

```suggestion
    def fit(self, train_df,unused=None):
```